resources/ec2-egress-only-internet-gateways.go

```diff
@@ -0,0 +1,60 @@
+package resources
+
+import (
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/rebuy-de/aws-nuke/pkg/types"
+)
+
+type EC2EgressOnlyInternetGateway struct {
+	svc *ec2.EC2
+	igw *ec2.EgressOnlyInternetGateway
+}
+
+func init() {
+	register("EC2EgressOnlyInternetGateway", ListEC2EgressOnlyInternetGateways)
+}
+
+func ListEC2EgressOnlyInternetGateways(sess *session.Session) ([]Resource, error) {
+	svc := ec2.New(sess)
+
+	resp, err := svc.DescribeEgressOnlyInternetGateways(nil)
```

Comment:
DescribeEgressOnlyInternetGateways could return more than one page. I think this i missing the token/nextToken handling like implemented here: https://github.com/rebuy-de/aws-nuke/blob/38b4ff979f1962f67caeeca011a34351ad640d33/resources/eks-fargate.go#L31-L47

Reply:
Thank you for the super fast feedback. I adjusted the PR accordingly. I was blindly using the structure from the existing _EC2InternetGateway_ resource. That one might be adjusted as well in a separate PR.

Reply:
> I was blindly using the structure from the existing EC2InternetGateway resource. That one might be adjusted as well in a separate PR.

This would be awesome, thanks.